code/modules/urist/modules/shipbattles/weapons.dm

```diff
@@ -277,6 +285,7 @@
 	loaded = 1
 	icon = 'icons/urist/items/ship_projectiles48x48.dmi'
 	icon_state = "bigtorpedo"
+	warhead = /obj/item/shipweapons/torpedo_warhead
```

Comment:
This will probably runtime when someone tries to remove the warhead. You will likely need to set this as `warhead = new /obj/item/shipweapons/torpedo_warhead` in the New() of the loaded torpedo.

Reply:
Worked fine when I tested it, plus it’s how stunbatons do it if memory serves, which is what I looked at for an example.

Reply:
Right, I’m dumb and misunderstood what you said, but still already handled. New() already has something for it, which is what I was talking about with the stunbaton example.